src/vs/workbench/parts/preferences/browser/settingsEditor2.ts

```diff
@@ -367,7 +367,7 @@ export class SettingsEditor2 extends BaseEditor {
 			if (foregroundColor) {
 				// Links appear inside other elements in markdown. CSS opacity acts like a mask. So we have to dynamically compute the description color to avoid
 				// applying an opacity to the link color.
-				const fgWithOpacity = new Color(new RGBA(foregroundColor.rgba.r, foregroundColor.rgba.g, foregroundColor.rgba.b, .7));
+				const fgWithOpacity = new Color(new RGBA(foregroundColor.rgba.r, foregroundColor.rgba.g, foregroundColor.rgba.b, .9));
```

Comment:
@roblourens FYI: I also updated the opacity for the Settings description text here (this was missing in my initial pr).